reverse.js

```diff
@@ -0,0 +1,12 @@
+let arr = [1, 2, 3, 6, 4];
+
+function reverse (arr){
+  let newArr = [];
```

Comment:
use const instead of let if you don't plan to change the value of the variable
```suggestion
  const newArr = [];
```